test/fixtures_identity_store.js

```diff
@@ -0,0 +1,181 @@
+// Identity Personas
+
+// NURSECONNECT
+    // (+27820001001) - new identity; no subscriptions
+    // (+27820001002) - existing identity with an active MomConnect subscription (no active NurseConnect subscription)
+    // (+27820001003) - existing identity with an active NurseConnect subscription
+
+
+module.exports = function() {
+    return [
+
+        // 0: get identity by msisdn +27820001001
```

Comment:
@gsvr could change fixture numbering to be in 100's.. thought that 1000's might be more 'future-proof' in some way...